<launch>
    <param name="scan_line" type="int" value="64" />
    <param name="mapping_skip_frame" type="int" value="1" />
    <param name="minimum_range" type="double" value="5"/>
    <param name="mapping_line_resolution" type="double" value="0.4"/>
    <param name="mapping_plane_resolution" type="double" value="0.8"/>

    <node pkg="fusion_localization_zc" type="laser_regist" name="laser_registration" output="screen" />
    <node pkg="fusion_localization_zc" type="laser_odo" name="LiDAROdometry" output="screen" />
    <param name="pub_wheel" type="bool" value="true" />
    <param name="saveLaserOdoINI" type="bool" value="true" />
    <param name="saveWheelOdo" type="bool" value="true" />
    <param name="USE_WHEEL" type="bool" value="false" />
    <node pkg="fusion_localization_zc" type="laser_map" name="laser_mapping" output="screen" />

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" output="log" args="-d $(find fusion_localization)/config/lidar_64.rviz" />
    </group>
</launch>

